### ISSUE TITLE
Sorting particular column in UI grid.

### ISSUE THREAD

**Issue by kalyanidesai** [angular-ui/ui-grid#6310]
Hello I am using Angular UI Grid to display the data in neat and clean manner. In the UI grid, there is one column as "Duration". I want to sort this column in descending order but unfortunately, i am unable to do that. I tried but it is sorting in text format. 
**In the controller part I have written as follows:**
 $scope.sortTrendingEvents = function () {
            $scope.slaGridOptions.data = $filter('orderBy')($scope.slaGridOptions.data, "duration", true)
            console.log('this button is clicked when trending event is pressed');
        };
In the HTML part I have written as:
<div class="row">
<button id="toggleFiltering" class="nxn-btn-primary nxn-mar-right-10" ng-click="sortTrendingEvents()">Trending Events</button>
</div>

In short what I have done is on the click of a button the Duration column must be sorted in DESC order but with the help of this code, 

I am unable to sort this column in number pattern. 
Please do guide me. Thank you so much.

**Comment by deeppatidar** [comment]
@kalyanidesai : I think you have to convert the row's data into integer before passing/sorting,
Because grid treats all rows data as text type by default.
